gpslab/geoip2/2.0/config/packages/gpslab_geoip.yaml

```diff
@@ -0,0 +1,8 @@
+# Read the README file for more detailed configuration information
+# https://github.com/gpslab/geoip2
+#gpslab_geoip:
+    # You should specify your personal licence key
+    #license: ''
```

Comment:
You should comment this and use an environment variable instead.

Reply:
The problem is that the user configures these parameters. The values for these parameters are individual for each user ([see docs](https://github.com/gpslab/geoip2#configuration)). It is not possible to fill these parameters with some predefined data. If you leave these parameters empty, an [error](https://travis-ci.org/symfony/recipes-contrib/jobs/646926681?utm_medium=notification&utm_source=github_status) occurs when building the project. I wrote about this problem [earlier](https://github.com/symfony/recipes-contrib/pull/837#issuecomment-582985103).

I see no other way to solve the problem than to add a commented out config.

Reply:
@Nyholm i can add environment variables, but without setting the default values ​​for them, the project will still not be builded.

```json
{
    "env": {
        "GEOIP_LICENSE": "",
        "GEOIP_EDITION": "GeoLite2-City"
    }
}
```

```yml
#gpslab_geoip:
    # You should specify your personal licence key
    #license: '%env(GEOIP_LICENSE)%'

    # Edition ID of database you need
    #edition: '%env(GEOIP_EDITION)%'
```

Reply:
Hm.. I would really prefer environment variables. 

How come the project cant be built?

----

I usually reject recipes with only config in comments. But this makes a lot of sense to add config and environment variables.

Reply:
> How come the project cant be built?

I wrote about this problem [earlier](https://github.com/symfony/recipes-contrib/pull/837#issuecomment-582985103).

In order to always have the current version of the GeoIP database, we update it with the composer install/update. This is convenient for quick start of development and quick deployment of the project. But at the first installation of the package, there are no required parameters for installing the GeoIP database. *This was not relevant before, but [MaxMind](https://www.maxmind.com/en/home) changed the policy of using his database.*

Reply:
@Nyholm the ideal config should look like this, but it will [not work](https://travis-ci.org/github/symfony/recipes-contrib/jobs/661017457#L326) because the license key is a required option.

```yml
gpslab_geoip:
    license: '%env(GEOIP_LICENSE)%'
    edition: 'GeoLite2-City'
```

```
# ./bin/console debug:config gpslab_geoip

Fatal error: Uncaught InvalidArgumentException: License for downloaded databases "default" is not specified.
```